backend/clubs/management/commands/deactivate.py

```diff
@@ -0,0 +1,29 @@
+from django.core.management.base import BaseCommand
+
+from clubs.models import Club
+
+
+class Command(BaseCommand):
+    help = "Deactivates all clubs in the database. This should be used at \
+            the end of the school year when clubs must be renewed."
+
+    def handle(self, *args, **kwargs):
+        print(
```

Comment:
You want to use `self.stdout` instead of `print` when writing management commands. From the [Django documentation](https://docs.djangoproject.com/en/3.0/howto/custom-management-commands/):

![image](https://user-images.githubusercontent.com/4441174/89845505-a9bdb100-db4c-11ea-93f7-e3f8d7de75c3.png)

Reply:
@cphalen I fixed this for you, see the updated file for my changes.

I also used `.update(active=True)` instead of the for loop. The difference between the two is that the for loop runs the following SQL queries:
```
UPDATE TABLE clubs SET active = True WHERE code = 'penn-labs';
UPDATE TABLE clubs SET active = True WHERE code = 'club-one';
UPDATE TABLE clubs SET active = True WHERE code = 'club-two';
...
```
The `.update` solution runs the following query:
```
UPDATE TABLE clubs SET active = True;
```
Since we're making one query and doing it on the SQL side, the update is much faster. It doesn't really matter for this use case, since we're only running it once per year, but I thought you might want to know!